### DISCUSSION TITLE
setValue not work with phoneNumber input

### DISCUSSION THREAD

**Discussion by sarashid** [typesnippet/antd-phone-input#113] [Q&A]
Hello 
I'm trying to update phoneNumber by setValue form already updated but number not shown in input how I fix it ?

**Comment by ArtyomVancyan** [comment]
@sarashid, I don't understand what `setValue` you're referring to. If you're referring to `setFieldValue` then the following function works fine:

```javascript
import {useCallback} from "react";
import {useForm} from "antd/es/form/Form";
import Form from "antd/es/form";
import Button from "antd/es/button";
import FormItem from "antd/es/form/FormItem";

const Demo = () => {
  const [form] = useForm();
  
  const updateValue = useCallback(() => {
      form.setFieldValue("phone", "+1 123 456 7890");
  }, [form])
  
  return (
    <Form form={form}>
        <FormItem name="phone">
            <PhoneInput />
        </FormItem>
        <Button onClick={updateValue}>Update Value</Button>
    </Form>
  )
}
```

**NOTE**: The first argument of `form.setFieldValue` must be the same as `name="phone"` of `FormItem`. Tested with the latest version of Ant Design (5.22.6)

If by `setValue` you meant state setter, then please look at [this example](https://github.com/typesnippet/antd-phone-input/tree/master/examples/antd5.x), compare your code with the sample, and find what you did wrong.
